frameworks/digital-outcomes-and-specialists/messages/homepage-sidebar.yml

```diff
@@ -30,7 +33,7 @@ open:
     You can apply to provide:
 
       - digital outcomes,
-        eg a discovery phase or an online billing application
+        eg a discovery phase for an online billing application
```

Comment:
I'm not sure this makes any more sense with the change. 'Online billing application' as an example is the problem. It should be changed to something that has a clearer sense of 'government service'.